SU2_CFD/src/output/CFlowOutput.cpp

```diff
@@ -784,6 +784,483 @@ void CFlowOutput::Set_CpInverseDesign(CSolver *solver, const CGeometry *geometry
 
 }
 
+void CFlowOutput::Add_NearfieldInverseDesignOutput(){
+
+  AddHistoryOutput("EQUIVALENT_AREA",   "CEquiv_Area",  ScreenOutputFormat::SCIENTIFIC, "EQUIVALENT_AREA", "Equivalent area", HistoryFieldType::COEFFICIENT);
+  
+}
+
+void CFlowOutput::Set_NearfieldInverseDesign(CSolver *solver, const CGeometry *geometry, const CConfig *config){
+
+  ofstream EquivArea_file, FuncGrad_file;
+  unsigned short iMarker = 0;
+  su2double auxXCoord, auxYCoord, auxZCoord, InverseDesign = 0.0, DeltaX,
+    Coord_i, Coord_j, jp1Coord, *Coord = nullptr, MeanFunction,
+    *Face_Normal = nullptr, auxArea, auxPress, jFunction, jp1Function;
+  unsigned long jVertex, iVertex, iPoint, nVertex_NearField = 0, auxPoint,
+    auxDomain;
+  unsigned short iPhiAngle;
```

Comment:
```suggestion
  su2double auxXCoord, auxYCoord, auxZCoord, InverseDesign, DeltaX,
    Coord_i, Coord_j, jp1Coord, *Coord = nullptr, MeanFunction,
    *Face_Normal = nullptr, auxArea, auxPress, jFunction, jp1Function;
  unsigned long iPoint, auxPoint, auxDomain;
```

Reply:
I am somewhat certain that removing iMarker, i,jVertex should be fine with the below suggested changes. But I wouldnt bet on myself here.